src/test/functional/delete-user-test.ts

```diff
@@ -0,0 +1,154 @@
+import {
+  createUserWithRoles, createAssignableRoles, assignRolesToParentRole,
+} from './shared/testingSupportApi';
+import {config as testConfig} from '../config';
+import * as Assert from 'assert';
+import {randomData} from './shared/random-data';
+import {BETA_FEATURES} from '../../main/app/feature-flags/flags';
+
+Feature('Delete User');
+
+const PARENT_ROLE = testConfig.TEST_SUITE_PREFIX + randomData.getRandomString(10);
+const ASSIGNABLE_CHILD_ROLE = testConfig.TEST_SUITE_PREFIX + randomData.getRandomString(10);
+const INDEPENDANT_CHILD_ROLE = testConfig.TEST_SUITE_PREFIX + randomData.getRandomString(10);
```

Comment:
Probably something small. random.getRandomString(0 is called a few times here and some other classes too. Might be better to create a utility method called createData() or something like that?

Reply:
@ashwini-mv, we can address this in a future PR? :)